mods/SparklesRolf@Furlatro/meta.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "Furlatro",
+  "requires-steamodded": true,
+  "requires-talisman": true,
+  "categories": ["Content", "Joker"],
+  "author": "SparklesRolf",
+  "repo": "https://github.com/SparklesRolf/Furlatro",
+  "downloadURL": "https://github.com/SparklesRolf/Furlatro/releases/latest/Furlatro.zip",

```

```suggestion
  "downloadURL": "https://github.com/SparklesRolf/Furlatro/releases/latest/download/Furlatro.zip",

```

the provided downloadURL is invalid, should be this instead

---

sorry about that, just committed the change now!